starter-code/apple-pie-starter-code/css/main.css

```diff
@@ -0,0 +1,43 @@
+:root {
+    text-align: center;
```

Comment:
Please remove this directive
It is bad practice to align text a global level in a root tag!